### PR TITLE
Bigtable read/write samples + quickstart

### PR DESCRIPTION
**Pull request by billyjacobson**

### CODE REVIEW

**Comment by billyjacobson** [comment]
@juan-rael please very consent in this PR

File: bigtable/api/src/hw_scan_all.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+// [START bigtable_hw_scan_all]
+
+use Google\Cloud\Bigtable\Admin\V2\StorageType;
+use Google\Cloud\Bigtable\Admin\V2\Cluster;
+use Google\Cloud\Bigtable\BigtableClient;
+use Google\Cloud\Bigtable\Mutations;
+use Google\ApiCore\ApiException;
+use Google\Cloud\Bigtable\Table;
+
+
```
Comment (bshaffer) [inline bigtable/api/src/hw_scan_all.php]: Remove extra newline!

File: bigtable/api/src/hw_scan_all.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+/** Uncomment and populate these variables in your code */
+// $project_id = 'The Google project ID';
+// $instance_id = 'The Bigtable instance ID';
+// $cluster_id = 'The Bigtable cluster ID';
+// $table_id = 'The Bigtable table ID';
+// $location_id = 'The Bigtable region ID';
+
+
```
Comment (bshaffer) [inline bigtable/api/src/hw_scan_all.php]: Remove this extra newline too!

File: bigtable/api/src/quickstart.php [hunk 3]
```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * Copyright 2018 Google LLC.
```
Comment (bshaffer) [inline bigtable/api/src/quickstart.php]: These should be 2019

File: bigtable/api/src/hw_create_table.php [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+// Include Google Cloud dependendencies using Composer
+require_once __DIR__ . '/../vendor/autoload.php';
+
+if (count($argv) != 4) {
+    return printf("Usage: php %s PROJECT_ID INSTANCE_ID TABLE_ID" . PHP_EOL, __FILE__);
+}
+list($_, $project_id, $instance_id, $table_id) = $argv;
+
+// [START bigtable_hw_create_table]
```
Comment (bshaffer) [inline bigtable/api/src/hw_create_table.php]: what is `hw_`?

File: bigtable/api/src/hw_get_with_filter.php [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+use Google\Cloud\Bigtable\Admin\V2\StorageType;
+use Google\ApiCore\ApiException;
+use Google\Cloud\Bigtable\V2\RowFilter;
+
+/** Uncomment and populate these variables in your code */
+// $project_id = 'The Google project ID';
+// $instance_id = 'The Bigtable instance ID';
+// $cluster_id = 'The Bigtable cluster ID';
+// $table_id = 'The Bigtable table ID';
+// $location_id = 'The Bigtable region ID';
```
Comment (bshaffer) [inline bigtable/api/src/hw_get_with_filter.php]: this is never used

File: bigtable/api/src/hw_get_with_filter.php [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+use Google\Cloud\Bigtable\Table;
+use Google\Cloud\Bigtable\Admin\V2\Cluster;
+use Google\Cloud\Bigtable\Admin\V2\StorageType;
+use Google\ApiCore\ApiException;
+use Google\Cloud\Bigtable\V2\RowFilter;
+
+/** Uncomment and populate these variables in your code */
+// $project_id = 'The Google project ID';
+// $instance_id = 'The Bigtable instance ID';
+// $cluster_id = 'The Bigtable cluster ID';
```
Comment (bshaffer) [inline bigtable/api/src/hw_get_with_filter.php]: this is never used

File: bigtable/api/src/hw_scan_all.php [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+use Google\ApiCore\ApiException;
+use Google\Cloud\Bigtable\Table;
+
+
+/** Uncomment and populate these variables in your code */
+// $project_id = 'The Google project ID';
+// $instance_id = 'The Bigtable instance ID';
+// $cluster_id = 'The Bigtable cluster ID';
+// $table_id = 'The Bigtable table ID';
+// $location_id = 'The Bigtable region ID';
```
Comment (bshaffer) [inline bigtable/api/src/hw_scan_all.php]: this is never used

File: bigtable/api/src/hw_write_rows.php [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+use Google\ApiCore\ApiException;
+use Google\Cloud\Bigtable\Table;
+
+
+/** Uncomment and populate these variables in your code */
+// $project_id = 'The Google project ID';
+// $instance_id = 'The Bigtable instance ID';
+// $cluster_id = 'The Bigtable cluster ID';
+// $table_id = 'The Bigtable table ID';
+// $location_id = 'The Bigtable region ID';
```
Comment (bshaffer) [inline bigtable/api/src/hw_write_rows.php]: this is never used

File: bigtable/api/src/hw_scan_all.php [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+use Google\Cloud\Bigtable\BigtableClient;
+use Google\Cloud\Bigtable\Mutations;
+use Google\ApiCore\ApiException;
+use Google\Cloud\Bigtable\Table;
+
+
+/** Uncomment and populate these variables in your code */
+// $project_id = 'The Google project ID';
+// $instance_id = 'The Bigtable instance ID';
+// $cluster_id = 'The Bigtable cluster ID';
```
Comment (bshaffer) [inline bigtable/api/src/hw_scan_all.php]: this is never used

File: bigtable/api/src/quickstart.php [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+use Google\Cloud\Bigtable\Table;
+use Google\Cloud\Bigtable\Admin\V2\Cluster;
+use Google\Cloud\Bigtable\Admin\V2\StorageType;
+use Google\ApiCore\ApiException;
+
+
+/** Uncomment and populate these variables in your code */
+// $project_id = 'The Google project ID';
+// $instance_id = 'The Bigtable instance ID';
+// $cluster_id = 'The Bigtable cluster ID';
```
Comment (bshaffer) [inline bigtable/api/src/quickstart.php]: this is never used

File: bigtable/api/src/quickstart.php [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+// $table_id = 'The Bigtable table ID';
+// $location_id = 'The Bigtable region ID';
```
Comment (bshaffer) [inline bigtable/api/src/quickstart.php]: this is neer used

File: bigtable/api/src/hw_write_rows.php [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+use Google\Cloud\Bigtable\BigtableClient;
+use Google\Cloud\Bigtable\Mutations;
+use Google\ApiCore\ApiException;
+use Google\Cloud\Bigtable\Table;
+
+
+/** Uncomment and populate these variables in your code */
+// $project_id = 'The Google project ID';
+// $instance_id = 'The Bigtable instance ID';
+// $cluster_id = 'The Bigtable cluster ID';
```
Comment (bshaffer) [inline bigtable/api/src/hw_write_rows.php]: this is never used

File: bigtable/api/src/hw_create_table.php (hunk 4 above)
Comment (billyjacobson) [reply bigtable/api/src/hw_create_table.php]: hw is hello world. I'm in the process of reorganizing all the bigtable samples and am using this as a prefix to know what it is used for

**Comment by juan-rael** [comment]
@billyjacobson do you have the consent url, to check it?

**Pull request merged**